File: Guanabara - PHP/exercicios-PHPOO/Aula12-Polimorfismo/Lobo.php
<?php
include_once 'Mamifero.php';
class Lobo extends Mamifero
{
    //Método implementado usando polimorfismo de sobreposição
    public function emitirSom() {
        echo '<p>Lobo uivando: AUUUUuuuuuuuuuuuuuu!</p>';
    }
}

File: Java/Meus exercícios POO/Aula13.Polimorfismo.Sobrecarga/src/main/java/sobrecarga/Lobo.java
package sobrecarga;
class Lobo extends Mamifero {
    //Polimorfismo por Sobreposição
    @Override //sobrescrevendo o método da classe `Mamífero`
    public void emitirSom() {
        System.out.println("Aauuuuuuuuuu!");
    }
}
